lib/rubocop/cop/lint/debugger.rb

```diff
@@ -35,6 +37,11 @@ module Lint
       class Debugger < Cop
         MSG = 'Remove debugger entry point `%<source>s`.'
 
+        DEBUGGER_METHODS = %i[
+          debugger byebug remote_byebug pry remote_pry pry_remote console rescue
+          save_and_open_page save_and_open_screenshot save_screenshot irb
+        ].to_set.freeze
+
         def_node_matcher :kernel?, <<~PATTERN
```

Comment:
I want to make a release of `rubocop-ast` soon so we can use `#global_const?` :-)